packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -142,22 +142,30 @@ export default class {
     }
 
     return this._updateRemote(id, {
+      enabled,
       name,
-      url,
       options,
-      enabled,
+      speed,
+      url,
     })
   }
 
   @synchronized()
-  async _updateRemote(id, { url, ...props }) {
+  async _updateRemote(id, { url, speed, ...props }) {
     const remote = await this._getRemote(id)
 
     // url is handled separately to take care of obfuscated values
     if (typeof url === 'string') {
       remote.url = format(sensitiveValues.merge(parse(url), parse(remote.url)))
     }
 
+    if (speed !== undefined) {
+      remote.speed = JSON.stringify(speed)
+    }
+    if (typeof remote.speed === 'object') {
+      remote.speed = JSON.stringify(remote.speed)
+    }
```

Comment:
As we said when we talked about it, I want to keep an history of it.

Maybe the properties should be called `benchmarks` and contain:

```js
const benchmarks = [
  {
    timestamp: -11480625000, // ms
    readRate: 102038, // bytes per second
    writeRate: 78062, // bytes per second
  }
]
```